# request (empty)

---
# response
sensor_msgs/Image captured_image